crates/bevy_transform/src/transform_propagate_system.rs

```diff
@@ -78,19 +63,15 @@ mod test {
         }
 
         // Root entity
-        let parent = world.spawn((Translation::new(1.0, 0.0, 0.0), Transform::identity()));
+        let parent = world.spawn((Transform::from(Translation::new(1.0, 0.0, 0.0)), ())); //FIXME: shouldn't need () to be added
```

Comment:
I think you're looking for a single element tuple, but I'm not sure. `(Transform::from(Translation::new(1.0, 0.0, 0.0)),)`